### DISCUSSION TITLE
Difference between 'children' and 'items' in props for useSingleSelectListState

### DISCUSSION THREAD

**Discussion by bstro** [adobe/react-spectrum#2929] [Q&A] [answered]
Hi there,

I'm trying to build a component (see https://github.com/adobe/react-spectrum/issues/2859#issuecomment-1051835486) that allows for a selection of a single number. Im currently using a simple 'useState' to manage the state for this component, but I'm trying to learn as much of the React-ARIA api as I can. It seems like `useSingleSelectListState` might be a suitable hook.

I'm not quite sure how to use the hook, however. In particular, I can't quite tell the difference between the purpose of `items`, and how that is different from `children`. Would love some insight.

**Comment by devongovett** [comment, marked as answer]
I'd recommend reading this document about how our collection components work: https://react-spectrum.adobe.com/react-stately/collections.html.
